data/config-schema/src/main/resources/net/corda/schema/configuration/flow/1.0/corda.flow.json

```diff
@@ -11,12 +11,12 @@
       "type": "object",
       "default": {},
       "properties": {
-        "maxRetryAttempts": {
-          "description": "The maximum number of times Corda retries a transient error before failing the flow. A value of zero disables retries.",
+        "maxRetryWindowDuration": {
+          "description": "The duration in milliseconds after a transient error that Corda retries a flow before failing it. A value of zero disables retries.",
           "type": "integer",
           "minimum": 0,
-          "maximum": 1000,
-          "default": 5
+          "maximum": 2147483647,
+          "default": 120000
         },
```

Comment:
Isn't this considered an API breaking change, as the flow configuration itself is modifiable by clients?.

Reply:
Yeah it is - this should be in a new version of the schema.

Reply:
Yes, unless we add maxRetryWindowDuration in a non-breaking way (as an optional config, leaving in maxRetryAttempts, even though it's non-functional). 

I'll pull this change out into a new version of the corda.flow.json, I assume the new config will be version 1.1

Reply:
Looping in @simon-johnson-r3 for his opinion here, regarding https://r3-cev.atlassian.net/wiki/spaces/CB/pages/4564091182/Making+a+Config+Change

Reply:
After discussing with Simon, we can't make breaking changes to config schema such as removing config properties due to reasons highlighted in the wiki page. I've added the new property with a default thus a non-breaking change.